docker/Dockerfile

```diff
@@ -75,9 +76,12 @@ COPY $DOCKERDIR/app_dev.php /app/elkarbackup/web/app_dev.php
 ## Disable "Manage parameters" menu item
 COPY $DOCKERDIR/src/Builder.php /app/elkarbackup/src/Binovo/ElkarBackupBundle/Menu/Builder.php
 
+COPY $DOCKERDIR/envars.sh /app/elkarbackup/app/envars.sh
 ## Composer install
 RUN set -ex; \
       cd /app/elkarbackup; \
+      chmod +x app/envars.sh;\
+      source app/envars.sh; \
```

Comment:
If we make envars.sh executable, then don't source it, but execute? Otherwise don't make it executable and don't put the interpreter line (sehbang)